my.css

```diff
@@ -16,10 +20,23 @@ body {
   outline: 2px solid green;
 }
 
-.paragraph {
-  font-size:  16px;
+// issue1: nesting
+body {
+  .product {
+    .product-imag {
+      .paragraph {
+        font-size:  1rem;
+      }
+    }
+  }
 }
 
-.layout {
+// over specifiy, creates the same issues as nesting actually
+body .product .product-image .layout {
```

Comment:
This is actually very related to nesting selector. Such a over-specificity creates a binding to the DOM and it's creating a css statement with a specificity being too high `0 - 4 - 1` (4x class, 1 element)